websites/N/Nexon Logistics/presence.ts

```diff
@@ -0,0 +1,180 @@
+			presence.getSetting<boolean>("timestamp"),
+			presence.getSetting<boolean>("privacy"),
+		]),
+		pages: Record<string, PresenceData> = {
+			"/about": { details: "Viewing About Page" },
+			"/gallery": { details: "Viewing Gallery Page" },
+			"/events": { details: "Browsing Events Page" },
+			"/newsletter": { details: "Browsing Newsletter" },
+			"/rules": { details: "Reading the rules" },
+			"/PrivacyPolicy": { details: "Reading Privacy Policy" },
+			"/Terms&Condition": { details: "Reading Terms&Condition" },
+			"/leaderboard": { details: "Viewing Leaderboard" },
+		};
+
+	for (const [path, data] of Object.entries(pages)) {
+		if (document.location.pathname.includes(path))
+			presenceData = { ...presenceData, ...data };
+	}
+
+	switch (document.location.host) {
+		case "nexonlogistics.com":
+			switch (true) {
+				case document.location.pathname.includes("/newsletter/"):
+					presenceData.details = "Reading a newsletter";
+					presenceData.state = document.querySelector<HTMLHeadingElement>(
+						"div.container-fluid > h1"
+					).textContent;
+					presenceData.buttons = [{ label: "Read Post", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/convoy"):
+					presenceData.details = "Browsing Nexon Events";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h1")[1].textContent;
+					presenceData.buttons = [{ label: "View Event", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/register"):
+					presenceData.details = "Browsing Nexon Applications";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h1")[1].textContent;
+					presenceData.buttons = [
+						{ label: "View Application", url: document.URL },
+					];
+					break;
+				case document.location.pathname.includes("/index"):
+					presenceData.details = "Viewing Nexon Homepage";
+					presenceData.state =
+						document.querySelector<HTMLHeadingElement>("h1").textContent;
+					presenceData.buttons = [
+						{ label: "View Nexon Homepage", url: document.URL },
+					];
+					break;
+				case document.location.pathname === "/":
+					presenceData.details = "Browsing the homepage";
+			}
+			break;
+		case "hub.nexonlogistics.com":
+			switch (true) {
+				case document.location.pathname.includes("/profile/"):
+					presenceData.details = "Viewing a profile";
+					presenceData.state =
+						document.querySelector<HTMLHeadingElement>("h4").textContent;
+					presenceData.buttons = [{ label: "View Profile", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/jobs/"):
+					presenceData.details = "Viewing a job";
+					presenceData.state =
+						document.querySelector<HTMLHeadingElement>("h3").textContent;
+					presenceData.buttons = [{ label: "View Job", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/events"):
+					presenceData.details = "Viewing Calendar";
```

Comment:
Consistency with capitalisation
```suggestion
					presenceData.details = "Viewing calendar";
```